dask/array/routines.py

```diff
@@ -1566,7 +1566,7 @@ def corrcoef(x, y=None, rowvar=1):
     return (c / sqr_d) / sqr_d.T
 
 
-@implements(np.round, np.round_)
+@implements(np.round)
```

Comment:
Does this change behaviour in any way?

Reply:
Yes, it drops support for `numpy.round_(<dask array>)`.
`numpy.round_` is undocumented and untested, both in dask and in numpy, and I personally never heard of it before.

Reply:
Looks like these are only aliases, so shouldn't matter